tests/models/test_model.py

```diff
@@ -171,6 +171,13 @@ def test_model_log():
         assert Version(loaded_model.mlflow_version) == Version(mlflow.version.VERSION)
 
 
+def test_model_log_inactive_run_id(tmp_path):
+    experiment_id = mlflow.create_experiment("test", artifact_location=str(tmp_path))
+    run = mlflow.MlflowClient().create_run(experiment_id=experiment_id)
+    model_info = Model.log("some/path", TestFlavor, run_id=run.info.run_id)
```

Comment:
```suggestion
    model_info = Model.log("model", TestFlavor, run_id=run.info.run_id)
```

We'll disallow `/` in the model name. Let's use `model`.